src/searching.md

```diff
@@ -35,24 +34,24 @@ finds notes with neither "cat" nor "mouse".
 same as the above.
 
 `"a dog"`\
-finds notes with the exact sequence of characters "a dog" on them, such
+finds notes with the exact sequence of characters "a dog" in them, such
 as "atta dog", but not "dog a" or "adog".
 
 `-"a dog"`\
-finds notes without the exact phrase "a dog"
+finds notes without the following sequence of characters: "a dog".
 
 `d_g`\
-finds notes with d, &lt;a letter&gt;, g, like dog, dig, dug, and so on.
+finds notes with d, &lt;one character&gt;, g, like dog, dig, dug, and so on.
 
 `d*g`\
-finds notes with d, &lt;zero or more letters&gt;, g, like dg, dog, dung,
+finds notes with d, &lt;zero or more characters&gt;, g, like dg, dog, dung,
 etc.
 
 `w:dog`\
-search for "dog" on a word boundary - will match "dog", but not "doggy"
-or "underdog". Requires Anki 2.1.24+ or AnkiMobile 2.1.61+. Note that
-formatting changes may be interpreted as word boundaries e.g. searching
-for `w:exam` will match **exam**ple.
+searches for the word "dog" as opposed to a sequence of characters - will match "dog", but not "doggy"
```

Comment:
(Comment about "searches" above.)

If "word boundary" needs to be defined, it seems better to do that without removing the phrase itself, because it is somewhat of a term of art in search syntax. 
`search for "dog" on a word boundary (as a whole word, not a sequence of characters in another word) - will match`

Reply:
In line 53 we use the term "word boundaries" and later we slowly transition to more frequent use of this. I don't think we really need to define word boundaries here. It is bit hard to define, and I think translators will find this sentence easier to deal with than your suggestion.